treeherder/perf/outcome_checker.py

```diff
@@ -0,0 +1,68 @@
+from django.db.models import Q, QuerySet
+
+from enum import Enum
+from typing import List
+
+from treeherder.model.models import Job, Push
+from treeherder.perf.models import (
+    BackfillRecord,
+)
+
+class OutcomeStatus(Enum):
+    SUCCESSFUL = 1
+    FAILED = 2
+    IN_PROGRESS = 3
+
+#TODO delete this after soft launch lands (job_type will come from BackfillRecord)
+def _get_job_type(record):
```

Comment:
I will rename this to `get_job_type`. This fuction will be removed after we have the Soft Launch db chanches